### PR TITLE
Course screen

### PR DESCRIPTION
**Pull request by caseytakahashi**

### CODE REVIEW

File: components/Form/CreateCourseForm.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            options={{
+              'sun': 'Sunday',
+              'mon': 'Monday',
+              'tue': 'Tuesday',
+              'wed': 'Wednesday',
+              'thu': 'Thursday',
+              'fri': 'Friday',
+              'sat': 'Saturday',
+            }}
+            placeholder='Select Day'/>
```
Comment (loischang) [inline components/Form/CreateCourseForm.js]: Placeholder doesn't display on screen

File: components/Form/CreateCourseForm.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+          <DatePickerField
+            ref='end_date'
+            minimumDate={today}
+            maximumDate={maxDate}
+            mode='date'
+            placeholder='End Date'/>
+        </Form>
+        <Button
+          onPress={() => this.props.onCreateCourse(this.state.courseData)}
```
Comment (loischang) [inline components/Form/CreateCourseForm.js]: Add field validations

File: components/Form/CreateCourseForm.js (hunk 1 above)
Comment (caseytakahashi) [reply components/Form/CreateCourseForm.js]: screen shot 2017-10-23 at 6 34 41 pm

**Comment by loischang** [comment]
TY C:

**Pull request merged**